rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/520_fetch_fields.yml

```diff
@@ -127,19 +127,74 @@ fetch _seq_no via stored_fields:
   - match: { hits.hits.0._seq_no: null }
 
 ---
-fetch _seq_no via fields:
+#fetch _seq_no via fields:
+
+#  - do:
+#      catch: bad_request
+#      search:
+#        index: test
+#        body:
+#          fields: [ _seq_no ]
+
+#  - match: { status: 400 }
+#  - match: { error.root_cause.0.type: illegal_argument_exception }
+#  - match: { error.root_cause.0.reason: "error fetching [_seq_no]: Cannot fetch values for internal field [_seq_no]." }
```

Comment:
I have commented on this test because it fails for the `yamlRestCompatTest` execution. I tried some methods described [here](https://github.com/elastic/elasticsearch/tree/main/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test#skipping-tests), but nothing worked for me. When and if the pr gets approved, I'll come back to investigate how to make this test workable.